### PR TITLE
Fix: Author profile design review changes

### PR DESCRIPTION
**Pull request by aronshamash**
Design changes as per https://nidigitalsolutions.jira.com/browse/REPLAT-1208

## Mobile
 
- Change padding above journalist image to 30px
- Change padding below journalist bio to 40px in total
- Adjust padding below article list module to 20px in total (currently 20px margin and 10px padding so remove just need to 10px from somewhere)
- Change font size of twitter handle to 15px

### Before
screen shot 2018-01-11 at 16 04 52

### After 
screen shot 2018-01-11 at 16 02 02

## Desktop (Chrome) /Tablet

- Remove white keyline around journalist image (all breakpoints)
- Change journalist image to be 116px wide/high (desktop & tablet)
- Change padding above journalist image to 60px (Desktop & Tablet)
- Change padding below twitter handle to 8px (desktop & tablet)

### Before
screen shot 2018-01-11 at 16 04 25

### After 
screen shot 2018-01-11 at 16 02 21

### Before
screen shot 2018-01-15 at 09 53 58

### After 
screen shot 2018-01-15 at 09 49 42

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/bd819dadb10e851c

File: packages/author-head/author-head-container.base.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  WrapperComponent: View
+};
+
+AuthorHeadContainer.propTypes = {
+  style: TextPropTypesStyle,
+  children: PropTypes.oneOfType([
+    PropTypes.element,
+    PropTypes.arrayOf(PropTypes.element)
+  ]).isRequired,
+  WrapperComponent: PropTypes.func
```
Comment (colincclark) [inline packages/author-head/author-head-container.base.js]: should this be required?

**Comment by tuncaulubilge** [comment]
Good job! 💯 
Got only one comment, the same top margin for author head should be applied to the loading state, which is in author-profile/author-profile-author-head.js 
Right now the loading placeholder is misaligned for author profile, see author profile "with provider" in storybook

File: packages/author-head/author-head-container.base.js (hunk 1 above)
Comment (aronshamash) [reply packages/author-head/author-head-container.base.js]: no because on native it is not sent and it defaults to a `View`

**Comment by aronshamash** [comment]
@tuncaulubilge good catch. I'll add that to the PR now

**Comment by BernieSumption** [comment]
@aronshamash The author profile photo disappears on the mobile breakpoint on web, is that intentional?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/4fe4c520d0f3aaa6

**Comment by aronshamash** [comment]
@BernieSumption thanks. I fixed that now.

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/28caaa719af101b9

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/a2d4f5bb80805908

File: packages/author-head/author-head-container.base.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  container: {
+    width: "100%",
+    alignItems: "center",
+    flexDirection: "column",
+    paddingBottom: 40,
+    backgroundColor: "#F9F8F3"
+  }
+});
+
+const AuthorHeadContainer = ({ style, children, WrapperComponent }) => (
```
Comment (craigbilner) [inline packages/author-head/author-head-container.base.js]: does this function bring much to the party?

Comment (aronshamash) [reply packages/author-head/author-head-container.base.js]: it allows padding to be added above the author image by passing in the responsive component. Simple wrapping an existing element/component is will not work due to the background colour as you get white space above.

**Comment by aronshamash** [comment]
@craigbilner IE11 screenshots attached now.

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/32788eaae3335010

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/16a8d1a391696f93

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/8bcaefc2222b58cf

**Pull request merged**